interfaces/builtin/avahi_observe.go

```diff
@@ -0,0 +1,130 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import "github.com/snapcore/snapd/interfaces"
+
+const avahiObserveConnectedPlugAppArmor = `
+# Description: allows domain browsing, service browsing and sevice resolving
+
+#include <abstractions/dbus-strict>
+dbus (send)
+    bus=system
+    path=/
+    interface=org.freedesktop.DBus.Peer
+    member=Ping
+    peer=(name=org.freedesktop.Avahi,label=unconfined),
+
+dbus (send)
+    bus=system
+    path=/
+    interface=org.freedesktop.Avahi.Server
+    member=Get*
+    peer=(name=org.freedesktop.Avahi,label=unconfined),
+
+
+# These allows tampering with other snap's browsers, so don't autoconnect for
```

Comment:
It says `don't autoconnect` here but below it's setting `autoconnect: true` (and a comment that the declaration is used). Maybe mention this in this comment too?